packages/connection/legacy/class-jetpack-xmlrpc-server.php

```diff
@@ -735,9 +734,13 @@ public function disconnect_blog() {
 	/**
 	 * Unlink a user from WordPress.com
 	 *
-	 * This will fail if called by the Master User.
+	 * If $user_id is not informed, it will try to disconnect the current logged in user. This will fail if called by the Master User.
+	 *
+	 * If $user_id is informed, it will try to disconnect the informed user, even if it's the Master User.
+	 *
+	 * @param integer $user_id The user ID to disconnect from this site.
```

Comment:
Actually, here is what happens:

When the xmlrpc server receives the request and calls the respective callback, it passes whatever it receives in the body as arguments.

If the request was done with an integer $user_id, that's what the callback will get.

But if the request was done without any arguments, the callback will get an empty array.

So maybe it's best to default to `array()` instead of `null` and the parameter would be `int|array`. Also I should make this explicit in the docs